src/AddResult.re

```diff
@@ -28,7 +77,6 @@ let make = (~communityName: string, ~onResultAdded) => {
   let toggleExtraTime = () => setExtraTime(oldExtraTime => !oldExtraTime);
 
   let (date, setDate) = React.useState(_ => Js.Date.make());
-  let (isAddingResult, setIsAddingResult) = React.useState(_ => false);
 
   let resetState = () => {

```

Comment:
`resetState` seems to not reset the `playerPicker` any more.

Reply:
This turned out to **not** be a regression, but only happens when a "most used player" exists. Created https://github.com/Yakimych/relogify/issues/111